shared/src/metabase/mbql/schema.cljc

```diff
@@ -680,6 +692,11 @@
 (defclause ^{:requires-features #{:temporal-extract}} ^:sugar get-second
   datetime DateTimeExpressionArg)
 
+(defclause ^{:requires-features #{:convert-timezone}} convert-timezone
+  datetime DateTimeExpressionArg
+  to       TimezoneIds
+  from     (optional TimezoneIds))
```

Comment:
Can we add some documentation here on the expected behaviour of convert-timezone in each of the 4 scenarios?